src/material.cpp

```diff
@@ -1889,3 +2050,25 @@ pyne::Material pyne::Material::operator/ (double y) {
   // Overloads x / y
   return pyne::Material(comp, mass / y, density );
 }
+
+bool pyne::detect_nuclidelist(hid_t data_set, std::string& nucpath){
```

Comment:
Got this one out of the `Material` class but kept it in the file as it is related to `Material` but I wanted to be able to access it outside of the `Material` class